composer.json

```diff
@@ -29,11 +29,11 @@
         }
     },
     "require": {
-        "php": "^7.1",
+        "php": "^7.2",
         "ext-json": "*",
         "psr/http-message": "^1.0",
         "psr/http-client": "^1.0",
-        "guzzlehttp/guzzle": "^6.3"
+        "guzzlehttp/guzzle": "^7.0.1"
     },
     "require-dev": {
         "phpunit/phpunit": "^7.5|^8.0",
```

Comment:
PHPUnit will need to be "^9.0" for Laravel 8

Reply:
Thank you. I've merged